### PR TITLE
Gives the clown a megaphone.

### PR DESCRIPTION
**Pull request by harmonyn**
Idea courtesy of @TehZombehz who may be doing a sprite
- Gives the clown a megaphone, which starts in his starting backpack in place of the april fool's meme (sorry leo) that wasn't even showing up for some reason.
- Standardizes the normal megaphone's style to that of a loud command headset, instead of a custom size in between.
- Fixes a godawful typo in the define for the above.
- Adds myself to admins.txt.

clown megaphone

:cl:
rscadd: Honk! Nanotrasen's clowning and development department has invented the clown megaphone, standard in all clowning loadouts! Honk honk!
/:cl:

### CODE REVIEW

**Comment by Iamgoofball** [comment]
You can just have SKUB and the m3gaphone in there you know, you just can't put it back in if you take one out

**Comment by Core0verload** [comment]
Your :cl: is a bit broken.

**Comment by bgobandit** [comment]
a) the skub wasn't even showing up in backpacks when I tested it
b) this makes more sense in the clown backpack, why would a clown start with skub

**Comment by Iamgoofball** [comment]
:-1: ANTI-SKUB PROPAGANDA 

SKUB FOR LIFE

**Comment by bgobandit** [comment]
On request of Zombehz I have updated the sprite.

honk

File: interface/stylesheet.dm [hunk 1]
```diff
@@ -93,11 +93,12 @@ h1.alert, h2.alert		{color: #000000;}
 .papyrus				{font-family: "Papyrus", cursive, sans-serif;}
 .robot					{font-family: "Courier New", cursive, sans-serif;}
 
-.commmand_headset       {font-weight: bold; font-size: 3;}
+.command_headset       {font-weight: bold; font-size: 3; }
 .big					{font-size: 3;}
 .reallybig				{font-size: 4;}
 .greentext				{color: #00FF00;	font-size: 3;}
 .redtext				{color: #FF0000;	font-size: 3;}
+.clown					{color: #FF69Bf;	font-size: 3; font-family: "Comic Sans MS", cursive, sans-serif; font-weight: bold;}
```
Comment (GunHog) [inline interface/stylesheet.dm]: The clown shoudl talk like this at all times.

**Comment by PKPenguin321** [comment]
Muh skub

**Comment by RandomMarine** [comment]
Put the skub in one of the theater lockers.

That way the clown and mime can fight over their skub preference.

**Comment by bgobandit** [comment]
no can do, map freeze

**Comment by GunHog** [comment]
That should be over now, the turfs PR got merged.

**Comment by duncathan** [comment]
Freeze is over

**Comment by ike709** [comment]
skub is love skub is life

**Comment by optimumtact** [comment]
SKUB

File: code/modules/jobs/job_types/civilian.dm [hunk 2]
```diff
@@ -30,7 +30,7 @@ Clown
 		/obj/item/weapon/stamp/clown = 1,
 		/obj/item/weapon/reagent_containers/spray/waterflower = 1,
 		/obj/item/weapon/reagent_containers/food/snacks/grown/banana = 1,
-		/obj/item/weapon/skub = 1
```
Comment (optimumtact) [inline code/modules/jobs/job_types/civilian.dm]: Bring back SKUB

Comment (Iamgoofball) [inline code/modules/jobs/job_types/civilian.dm]: SKUB FOR LIFE

**Pull request merged**